deploy/scripts/setup_database_users.sh

```
@@ -0,0 +1,90 @@
+# Function to execute SQL file with environment variable substitution
+execute_sql_file() {
+    local sql_file="$1"
+    if [[ ! -f "$sql_file" ]]; then
+        echo "Error: SQL file not found: $sql_file"
+        exit 1
+    fi
+
+    echo "Executing SQL file: $sql_file"
+
+    # Create a temporary file with environment variable substitution
+    local temp_sql_file=$(mktemp)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)